server/src/http_server/pages/blog/md/html.rs

```diff
@@ -287,13 +291,25 @@ impl IntoHtml for Emphasis {
 }
 
 impl IntoHtml for Image {
-    fn into_html(
-        self,
-        _config: &AppConfig,
-        _context: &SyntaxHighlightingContext,
-    ) -> Result<Markup> {
+    fn into_html(self, config: &AppConfig, context: &SyntaxHighlightingContext) -> Result<Markup> {
```

Comment:
function `into_html` take the `Markdown` image syntax into HTML, constructing the `image URL` by combining the` base URL`, `article path`, and `image filename`.